File: Domain/Entities/Subscription.cs
namespace Domain.Entities
{
    public class Subscription
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public decimal Price { get; set; }

        public string Description { get; set; }

        public bool IsDefault { get; set; }

        public TimeSpan Duration { get; set; }

        public ICollection<AppUser> Subscribers { get; set; }
    }
}

File: client-app/src/app/models/subscription.ts
export interface Subscription{
    id: number,
    name: string,
    price: number,
    description: string,
    duration: Date,
    isDefault: boolean
}

export class Subscription implements Subscription {
    constructor(sub: Subscription) {
        this.id = sub.id;
        this.name = sub.name;
        this.price = sub.price;
        this.description = sub.description;
        this.duration = sub.duration,
        this.isDefault = sub.isDefault
    }
}